test/stdcpp/win64.mak

```diff
@@ -5,29 +5,34 @@ MODEL=64
 DRUNTIMELIB=druntime64.lib
 CC=cl
 
-_MSC_VER = $(file < ..\..\ver.txt)
+TESTS=array allocator new string
 
-TESTS= array allocator new string
+_MSC_VER=$(file < ..\..\ver.txt)
```

Comment:
I would be very surprised if this works with Digital Mars make.
AFAICT including a file only works with a line like this (from druntime/win64.mak):
```
$(mak\COPY)
```
This reads a single assignment from the file.

Reply:
What do you expect doesn't work?
This does work... the tests all pass.

Reply:
Look for `-version=_MSC_VER_xxxx` in the CI logs. That value is read from this line at your comment.

Reply:
Surprising. `$(file < MAK\COPY)` is empty for me. `_MSC_VER=$(..\..\ver.txt)` would work.

I suspect the auto-tester is not using Digital Mars make here?

Reply:
I ran the same script locally... It works, and the tests invoke with all proper args... How are you running it?
auto-tester and appveyor both seem to work... My local machine too. I'm surprised...

Reply:
Like, check the CI logs. The msc ver and the cflags and dflags are supplied correctly. It also did build `string_view`, which is pulled the same way...

Reply:
Your PR passes for me locally, too. I'm confused ;-)